Cargo.toml

```diff
@@ -83,11 +83,10 @@ error_code = { path = "components/error_code" }
 fail = "0.3"
 failure = "0.1"
 fs2 = "0.4"
-futures = "0.1"
-futures03 = { package = "futures", version = "0.3.1", features = ["compat", "thread-pool"] }
+futures = { version = "0.3", features = ["compat", "thread-pool"] }
```

Comment:
what do we need the `compat` feature for?

Reply:
fixed

Reply:
same as https://github.com/tikv/tikv/pull/8625#discussion_r486756752